### DISCUSSION TITLE
Please explain what icons in Dashboard mean.

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#1862] [General]
Hi. After recent upgrade to a much newer version we are seeing a lot of icons, some of which appear in red or green and also the background color of the row in the "Open chats" box can be while, gray or pink. The title of the chat tabs also has icons with different colors. There is this page https://livehelperchat.com/article/view/42 which could updated and maybe some icons can show meaning when pointer is over them. Thanks

**Comment by remdex** [comment]
Please read this section
https://doc.livehelperchat.com/docs/dashboard/#main-icons-in-the-widgets

**Comment by alonsoac** [comment]
Hi. There is a setting "How long we should wait before we inform operator about unanswered chat." and this appears to control the red clock icon and red background for Open chats box.  But the chats that are pending are not marked in any way.   Is there some config to mark pending chats in red after some time?

**Reply by remdex** [reply]
No,
This icon is applied only to the left menu box where you have opened chats. In all other cases you can see.

* Either pending time in pending widget
* Either last response time in red arrow which indicates how long time passed since visitor message.

**Reply by remdex** [reply]
You can also try to setup https://doc.livehelperchat.com/docs/development/webhooks#how-to-setup-a-continuous-hook-event and add icons based on dynamic conditions. E.g add icon if this time is long, just as you wanted.
